src/components/operations/operation-details/ko/runtime/operation-console.ts

```diff
@@ -373,12 +375,58 @@ export class OperationConsole {
         this.sendRequest();
     }
 
-    public async sendFromBrowser<T>(request: HttpRequest): Promise<HttpResponse<T>> {
-        const response = await this.httpClient.send<any>(request);
-        return response;
+    public async sendFromBrowser<T>(url: string, method: string, headers: ConsoleHeader[], body: any, operationName: string) {
+        const stringifiedHeaders: object = headers.map(header => `${header.name()}: ${header.value()}`);
+        const response = await fetch(url, {
+            method: method,
+            headers: { ...stringifiedHeaders },
+            body: body
+        });
+
+        let headersString = '';
+        response.headers.forEach((value, name) => headersString += `${name}: ${value}\n`);
+
+        const contentTypeHeaderValue = response.headers.get(KnownHttpHeaders.ContentType);
+
+        const responseReturn: any = {
+            headers: headersString,
+            contentTypeHeader: contentTypeHeaderValue,
+            statusCode: response.status,
+            statusText: response.statusText
+        };
+
+        if (downloadableTypes.some(type => contentTypeHeaderValue.includes(type))) {
+            const reader = response.body.getReader();
+            const chunks = [];
+
+            while (true) {
+                const { done, value } = await reader.read();
+          
+                if (done) {
+                    break;
+                }
+
+                chunks.push(value); 
+            }
+
+            const blob = new Blob(chunks, { type: contentTypeHeaderValue });
+            const fileExtension = getExtension(contentTypeHeaderValue);
+
+            if (fileExtension) {
+                saveAs(blob, operationName + '.' + fileExtension);
+            } else if (contentTypeHeaderValue.includes('zip')) {
```

Comment:
Yes, we absolutely need it because your specific example has Content-Type header "application/x-zip-compressed" which mime library does not recognize (returns null).

Reply:
I see. How about we define [additional mapping](https://github.com/broofa/mime#mimedefinetypemap-force--false) for mime library (let's say somewhere in the startup files)?

I believe it would be something like this:
`mime.define({'application/x-zip-compressed': ['zip']})`